docs/modules/ROOT/pages/how-to-guides/proc_release_application.adoc

```diff
@@ -165,22 +113,39 @@ metadata:
  namespace: managed <3>
 spec:
  application: demo-app <4>
- environment: sre-production <5>
- origin:
-   workspace: <user-workspace> <6>
-   namespace: <dev-workspace> <7>
- releaseStrategy: <release-strategy> <8>
+ data: <key> <5>
+ environment: sre-production <6>
+ origin: <dev-workspace> <7>
+ pipelineRef: <pipeline_ref> <8>
+ policy: <policy> <9>
+ serviceAccount: <service-account> <10>
+
 ----
 
 +
 <1> The name of the resource created by the Environment team.
```

Comment:
I don't follow this

Reply:
I've removed the reference.

Reply:
I still see this for the ReleasePlan:

1. The name of the resource that you are creating.
2. The name of the release plan.

But 1. is the `kind`. The note should be removed.